.idea/elephants_cse5911.iml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think this is an artifact from JetBrains. We should probably be gitignoring the .idea directory